README.md

```diff
@@ -1 +1,158 @@
+
+```shell
+cd my-folder
+git clone https://github.com/Zafron047/Capstone_Project.git
+```
+
+--->
+
+### Usage:
+
+To explore the tools I have used to create my portfolio, follow these steps:
+
+1. Open the project in Visual Studio Code (VSCode).
+2. Take a look at the project structure and files to understand the implementation.
+3. Review the code, HTML, CSS, and other resources to see how the portfolio is built.
+
+Feel free to explore and customize the project according to your needs.
+
+
+### Run tests:
+
+To execute the tests for this project, follow these steps:
+
+1. Open the project in Visual Studio Code (VSCode).
+2. Install the "Live Server" extension from the VSCode marketplace.
+3. Open the `index.html` or `about.html` file in VSCode.
+4. Right-click on the `index.html` or `about.html` file and select "Open with Live Server" from the context menu.
+5. This will launch the project in your default web browser and automatically refresh it when changes are made.
+6. Verify that the tests are running successfully and check the console for any error messages or test results.
+
+By following these steps, you can effectively run and validate the tests for the project using Live Server in VSCode.
+
+
+### Deployment:
+
+[Click to see Live demo.](https://zafron047.github.io/Capstone_Project/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **MD. Nohain Islam**
+
+- GitHub: [@githubhandle](https://github.com/Zafron047)
+- Twitter: [@twitterhandle](https://https://twitter.com/NohainZ)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/nohain-islam-0261b4148/)
+
+Both authors have contributed to the development of this project.
+
+## 🔭 Future Features <a name="future-features"></a>
+
+Here are some planned future features for this project:
+
+- **[React.js]**: Integrate React.js framework to enhance interactivity and component-based development.
+- **[Webpack]**: Utilize Webpack as a module bundler for efficient asset management and optimization.
+
+These features will further enhance the functionality and performance of the project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Zafron047/Capstone_Project/issues).
+
+<br>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you find this project helpful and valuable, please consider showing your support by giving it a ⭐️ rating!
+
+Your support and encouragement are highly appreciated and motivate me to continue working on projects like this.
+
+Thank you for your support!
+
+<br>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to express my sincere gratitude to Microverse for providing me with this opportunity to learn and grow as a developer. Their support and guidance have been invaluable throughout my journey.
```

Comment:
According to the project description, it is expected to give appropriate acknowledgements to the author of the design [Cindy Shin](https://www.behance.net/adagio07), in order not to come off as plagiarizing.